api/specs/web-server/_computations.py

```diff
@@ -33,7 +33,7 @@ async def get_computation(project_id: ProjectID):
             "description": "Project/wallet/pricing details not found"
         },
         status.HTTP_402_PAYMENT_REQUIRED: {
-            "description": "Insufficient osparc credits"
+            "description": "Insufficient osparc credits."
```

Comment:
a dot? ok... does it also need a dot in the other ones?

Reply:
This message will be shown on all products, I wouldn't mention "osparc".